pkg/cincinnati/cincinnati_test.go

```diff
@@ -132,7 +133,7 @@ func TestGetUpdates(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			updates, err := c.GetUpdates(uri, arch, channelName, semver.MustParse(test.version))
+			updates, err := c.GetUpdates(context.TODO(), uri, arch, channelName, semver.MustParse(test.version))
```

Comment:
tests can use [`context.Background()`][1].  `TODO` is for when you intend to go back later and provide a real `Context`, and for this case we are fine with no `TestGetUpdates`-side timeouts.

[1]: https://golang.org/pkg/context/#Background

Reply:
fixed